src/coinbase/staking_operation.ts

```diff
@@ -36,6 +42,34 @@ export class StakingOperation {
     return this.transactions;
   }
 
+  /**
+   * Get the status of the staking operation.
+   *
+   * @returns The status of the staking operation.
+   */
+  public getStatus(): StakingOperationStatusEnum {
+    return this.model.status;
+  }
+
+  /**
+   * Get the staking operation for the given ID.
+   *
+   * @returns The staking operation object.
+   */
+  public async get(): Promise<StakingOperationModel> {
```

Comment:
```suggestion
  public async fetch(): Promise<StakingOperationModel> {
```
(to match other methods)

Reply:
done